packages/delay-p/src/index.js

```diff
@@ -1,9 +1,9 @@
 import { CANCEL } from '@redux-saga/symbols'
 
-export default function delayP(ms) {
+export default function delayP(ms, returnValue = true) {
```

Comment:
imho shouldn't have default value

Reply:
I think the default value is fine. When we do not care the return value, we can omit the `returnValue`. And the default value (`true`) is useful when we use `delayP` in a race:

```javascript
const result= race({
  timeout: call(delayP, 2000),
  response: call(fetch, url),
})
// result.timeout is true if delayP resolves first
if (result.timeout) { /* ... */ }
else { /* ... */ }
```

Reply:
The behavior now is the same as in v0.16.1. I think this is a small mistake when separating internal modules. 
https://github.com/redux-saga/redux-saga/blob/600802fe2c91be57c6d0c659424b9844739b5379/src/internal/utils.js#L94-L103